modules/learn/pages/clusters-and-availability/groups.adoc

```diff
@@ -208,6 +214,8 @@ For example, given a cluster:
 
 At a minimum, one instance of the Index Service and one instance of the Search Service should be deployed on each rack.
 
+Also, for auto-failover to be possible, the service-specific auto-failover constraints be met -- the policy information is documented in xref:learn:clusters-and-availability/automatic-failover.adoc#failover-policy[Service-Specific Auto-Failover Policy] -- it lists the number of nodes that each service must be running on and explains the xref:learn:clusters-and-availability/automatic-failover.adoc#data-service-preference[Data Service Preference] when a service is co-located with the Data Service.
```

Comment:
Typo.
Instead of:
Also, for auto-failover to be possible, the service-specific auto-failover constraints be met
Should be:
Also, for auto-failover to be possible, the service-specific auto-failover constraints must be met